doc/source/io.rst

```diff
@@ -3884,28 +3885,54 @@ destination DataFrame as well as a preferred column order as follows:
                              index_col='index_column_name',
                              col_order=['col1', 'col2', 'col3'], project_id = projectid)
 
-Finally, you can append data to a BigQuery table from a pandas DataFrame
-using the :func:`~pandas.io.to_gbq` function. This function uses the
-Google streaming API which requires that your destination table exists in
-BigQuery. Given the BigQuery table already exists, your DataFrame should
-match the destination table in column order, structure, and data types.
-DataFrame indexes are not supported. By default, rows are streamed to
-BigQuery in chunks of 10,000 rows, but you can pass other chuck values
-via the ``chunksize`` argument. You can also see the progess of your
-post via the ``verbose`` flag which defaults to ``True``. The http
-response code of Google BigQuery can be successful (200) even if the
-append failed. For this reason, if there is a failure to append to the
-table, the complete error response from BigQuery is returned which
-can be quite long given it provides a status for each row. You may want
-to start with smaller chunks to test that the size and types of your
-dataframe match your destination table to make debugging simpler.
+By default, the query result is returned as a dataframe, but you can redirect the results
+to a destination table via the ``destination_table`` argument. Use the ``if_exists`` argument
+to specify the action to take when the destination table already exists. For example, the following
+query will insert the results of the query directly to a bigquery table, instead of returning a dataframe:
+
+.. code-block:: python
+
+   data_frame = pd.read_gbq('SELECT * FROM test_dataset.test_table',
+                             index_col='index_column_name',
+                             col_order=['col1', 'col2', 'col3'], project_id = projectid,
```

Comment:
for sure create a new pr
but not sure why you actually need a new function; why wouldn't  this just be part of to_gbq?